acquisitions/src/main/resources/thunderjet/mod-orders/features/MODORDERS-538-piece-against-non-package-mixed-pol-manual-piece-creation-is-true.feature

```diff
@@ -224,7 +224,7 @@ Feature: Should create and delete pieces for non package mixed POL with quantity
     * def itemIdWithItemAndHolding = karate.jsonPath(response, '$.items[*][?(@.enumeration == "' + pieceIdWithItemAndHolding + '")]')
     And match itemIdWithItemAndHolding[0].enumeration == '#(pieceIdWithItemAndHolding)'
     And match itemIdWithItemAndHolding[0].chronology == '#(pieceIdWithItemAndHolding)'
-    And match itemIdWithItemAndHolding[0].discoverySuppress == true
+    And match itemIdWithItemAndHolding[0].discoverySuppress == null
```

Comment:
relates to the latest PR for https://folio-org.atlassian.net/browse/MODORDERS-1316